### PR TITLE
Add `KHTML` and `etag`

### PR DESCRIPTION
**Pull request by kenji-miyake**

### CODE REVIEW

**Comment by kmiya** [comment]
Thanks! But the words have not been sorted by GitHub Actions. I'll check that.

File: cspell/.cspell.json [hunk 1]
```diff
@@ -22,6 +22,8 @@
         "reports*/**"
     ],
     "words": [
+        "KHTML",
+        "etag"
```
Comment (kmiya) [inline cspell/.cspell.json:26]:
```suggestion
        "etag",
```

Comment (kenji-miyake) [reply cspell/.cspell.json:26]: Ah, sorry :pleading_face:

Comment (kenji-miyake) [reply cspell/.cspell.json]: I'll update the script to check this error.

**Review by kmiya — APPROVED** [review]
Thanks!
Memo: the words come from SVG file format.

**Pull request merged**